test/test-operation.js

```diff
@@ -32,260 +32,638 @@ var helpers = require('./helpers');
 var Sway = helpers.getSway();
 
 describe('Operation', function () {
-  var swaggerApi;
+  context('swaggerApiRelativeRefs', function () {
```

Comment:
It's not strictly necessary but I added it because I [had to supply `jsonRefs` manually](https://github.com/apigee-127/sway/pull/117/files#diff-42a353e98c906af524c7d3d48feaf72aR75) to get the local refs parsed correctly.
Also, looking at [the test you linked](https://github.com/apigee-127/sway/blob/e53e529dc4a01b11a40f9e968fc15be76a292c73/test/test-issues.js#L69), I think this test is actually nor working. The result of `Swagger.create` appears to not be used at all and the test is really testing the object from [line 39](https://github.com/apigee-127/sway/blob/e53e529dc4a01b11a40f9e968fc15be76a292c73/test/test-issues.js#L39).

To sum it up, I'm happy to remove the helper but after considering the above, I'd suggest to use the helper in `test-issues.js` too.